src/named.rs

```diff
@@ -280,8 +280,8 @@ pub enum NamedChain {
 
     Elastos = 20,
 
-    #[cfg_attr(feature = "serde", serde(alias = "kakarot-sepolia"))]
-    KakarotSepolia = 1802203764,
+    #[cfg_attr(feature = "serde", serde(alias = "kakarot-starknet-sepolia"))]
```

Comment:
can we add the previous alias as an additional serde alias?

Reply:
instead of a breaking change we should have both, and the enum variant should be kept the same

Reply:
Updated the code based on the comments. Please review.